README.md

```diff
@@ -53,9 +53,12 @@ https://firebase.google.com.
 ## Getting Started
 
 * Install the latest Android Studio (should be 3.0.1 or later)
-* Clone the repo (`git clone git@github.com:firebase/firebase-android-sdk.git`)
+* Clone the repo (`git clone --recurse-submodules --remote-submodules git@github.com:firebase/firebase-android-sdk.git`)
```

Comment:
We don't use `--remote-submodules` when syncing because it pulls the submodules from "head" in their respective repos, not from the exact commit we want to. 

Each project is responsible to update their remotes if deemed necessary.

Reply:
Done, thanks!